mt-static/plugins/TinyMCE/lib/js/adapter.js

```diff
@@ -10,10 +10,12 @@
     MT.Editor.TinyMCE = function() { MT.Editor.apply(this, arguments) };
 
     var suffix = '';
+    var cache_suffix = '';
     if(jQuery('script[src*="tinymce.min"]').length){
         // min
         suffix = '.min';
     }
+    cache_suffix = jQuery('script[src*="tinymce"]').attr("src").replace(/.*\?/, '');
```

Comment:
I would simply be better off declaring it here.

```suggestion
    var cache_suffix = jQuery('script[src*="tinymce"]').attr("src").replace(/.*\?/, '');
```